### DISCUSSION TITLE
Galaxy Brain helping gang!

### DISCUSSION THREAD

**Discussion by sleepymalc** [Schweinepriester/github-profile-achievements#134] [Q&A] [answered]
@thenithinbalaji Here!

**Comment by thenithinbalaji** [comment, marked as answer]
# How do you get Galaxy brain?
+ Answer at least 2 discussions. 
+ Take note, you cannot mark your own answers to your own questions.

![image](https://user-images.githubusercontent.com/73932121/179235281-69524f35-21a5-4c53-a89d-35dccc87ebf0.png)

# This badge not only has the base version, but also tiers.

![image](https://user-images.githubusercontent.com/73932121/179235882-033cffe0-4bbd-44e2-a3be-b1a078745b05.png)

**Comment by YSD98** [comment]
I want a galaxy brain Badge
